textacy/texts.py

```diff
@@ -92,6 +96,62 @@ def __iter__(self):
         for tok in self.spacy_doc:
             yield tok
 
+    def save(self, path, fname_prefix=None):
+        """
+        Save serialized TextDoc content and metadata to disk.
+
+        Args:
+            path (str): directory on disk where content + metadata will be saved
+            fname_prefix (str, optional): prepend standard filenames 'spacy_doc.bin'
+                and 'metadata.json' with additional identifying information
+        """
+        if fname_prefix:
+            meta_fname = os.path.join(path, '_'.join([fname_prefix, 'metadata.json']))
+            doc_fname = os.path.join(path, '_'.join([fname_prefix, 'spacy_doc.bin']))
+        else:
+            meta_fname = os.path.join(path, 'metadata.json')
+            doc_fname = os.path.join(path, 'spacy_doc.bin')
+        package_info = {'textacy_lang': self.lang,
+                        'spacy_version': spacy.about.__version__}
```

Comment:
I _think_ that's not needed. The file formats themselves don't depend on textacy, the only issue would be if I decided to change the structure or naming, which would throw errors rather than fail silently.